management/src/main/java/io/micronaut/management/endpoint/management/impl/ManagementResponse.java

```diff
@@ -0,0 +1,13 @@
+package io.micronaut.management.endpoint.management.impl;
+
+import io.micronaut.http.hateoas.AbstractResource;
+import io.micronaut.management.endpoint.management.ManagementEndpoint;
+
+/**
+ * <p>Allow {@link ManagementEndpoint} to provide a response in HATEOAS architecture</p>
+ *
+ * @author Hernán Cervera
+ * @since 2.5
+ */
+public class ManagementResponse extends AbstractResource<ManagementResponse> {
```

Comment:
Class name is a bit generic. Change to ManagementEndpointResponse

Reply:
I changed it to `ManagementControllerResponse`, since the class is now a controller, not an endpoint